launcher/modplatform/modpacksch/FTBPackInstallTask.cpp

```diff
@@ -343,4 +352,35 @@ void PackInstallTask::onModDownloadFailed(QString reason)
     emitFailed(reason);
 }
 
+/// @brief copy the matched blocked mods to the instance staging area
+void PackInstallTask::copyBlockedMods() {
+
+    setStatus(tr("Copying Blocked Mods..."));
+    setAbortable(false);
+    int i = 0;
+    int total = m_blocked_mods.length();
+    setProgress(i, total);
+    for (auto mod = m_blocked_mods.begin(); mod != m_blocked_mods.end(); mod++, i++) {
```

Comment:
a range-for loop would be nicer here i think

Reply:
done, I was being silly when I used it just to attach `i` to the loop.